docs/stories/QS-178.story_review_fix_#03.md

```
@@ -0,0 +1,1058 @@
+
+---
+
+### [should-fix] AC-6 story text describes closed-window decrement
+- File: `docs/stories/QS-178.story.md` AC-6 lines 145-148
+- Severity: should-fix
+- Source: qs-review-acceptance-auditor
+
+#### Description / Fix
+
+AC-6 still shows the closed-window pseudocode:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)